### PR TITLE
lightning: refine description for `block-size`

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):https://github.com/pingcap/docs-cn/pull/16231

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -203,7 +203,7 @@ addr = "172.16.31.10:8287"
 # 该参数自 v7.1.0 版本开始引入。
 # region-check-backoff-limit = 1800
 
-# 物理导入模式下，用于控制本地排序文件的 I/O 块大小。当 IOPS 成为瓶颈时，可以将该参数的值调大。
+# 物理导入模式下，用于控制本地文件排序的 I/O 块大小。当 IOPS 成为瓶颈时，你可以调大该参数的值以缓解 Disk IOPS，从而提升性能。
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-configuration.md:206]:
```suggestion
# 物理导入模式下，用于控制本地文件排序的 I/O 块大小。当 IOPS 成为瓶颈时，你可以调大该参数的值以缓解磁盘 IOPS，从而提升性能。
```

**Review by ran-huang — APPROVED** [review]

**Comment by lyzx2001** [comment]
need further check of expected behaviour with @frank945946

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -203,7 +203,7 @@ addr = "172.16.31.10:8287"
 # 该参数自 v7.1.0 版本开始引入。
 # region-check-backoff-limit = 1800
 
-# 物理导入模式下，用于控制本地排序文件的 I/O 块大小。当 IOPS 成为瓶颈时，可以将该参数的值调大。
+# 物理导入模式下，用于控制本地文件排序的 I/O 块大小。当 IOPS 成为瓶颈时，你可以调大该参数的值以缓解磁盘 IOPS，从而提升性能。
```
Comment (lyzx2001) [inline tidb-lightning/tidb-lightning-configuration.md:206]:
```suggestion
# 物理导入模式下，用于控制本地文件排序的 I/O 区块大小。当 IOPS 成为瓶颈时，你可以调大该参数的值以缓解磁盘 IOPS，从而提升导入性能。
```

**Review by lyzx2001 — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -203,8 +203,8 @@ addr = "172.16.31.10:8287"
 # 该参数自 v7.1.0 版本开始引入。
 # region-check-backoff-limit = 1800
 
-# 物理导入模式下，用于控制本地排序文件的 I/O 块大小。当 IOPS 成为瓶颈时，可以将该参数的值调大。
-# 该参数自 v7.6.0 版本开始引入。默认值为 "16KiB"。
+# 物理导入模式下，用于控制本地文件排序的 I/O 区块大小。当 IOPS 成为瓶颈时，你可以调大该参数的值以缓解磁盘 IOPS，从而提升数据导入性能。
+# 该参数自 v7.6.0 版本开始引入。默认值为 "16KiB"。注意，如果仅指定数字（如 `16`），则单位为 Byte 而不是 KiB。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:207]:
```suggestion
# 该参数自 v7.6.0 版本开始引入。默认值为 "16KiB"。取值必须大于或等于 `1`。注意，如果仅指定数字（如 `16`），则单位为 Byte 而不是 KiB。
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 4]
```diff
@@ -203,8 +203,8 @@ addr = "172.16.31.10:8287"
 # 该参数自 v7.1.0 版本开始引入。
 # region-check-backoff-limit = 1800
 
-# 物理导入模式下，用于控制本地排序文件的 I/O 块大小。当 IOPS 成为瓶颈时，可以将该参数的值调大。
-# 该参数自 v7.6.0 版本开始引入。默认值为 "16KiB"。
+# 物理导入模式下，用于控制本地文件排序的 I/O 区块大小。当 IOPS 成为瓶颈时，你可以调大该参数的值以缓解磁盘 IOPS，从而提升数据导入性能。
+# 该参数自 v7.6.0 版本开始引入。默认值为 "16KiB"。取值必须大于或等于 `1B`。注意，如果仅指定数字（如 `16`），则单位为 Byte 而不是 KiB。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:207]:
```suggestion
# 该参数自 v7.6.0 版本开始引入。默认值为 "16KiB"。取值必须为整数，且必须大于或等于 `1B`。注意，如果仅指定数字（如 `16`），则单位为 Byte 而不是 KiB。
```

**Pull request merged**